bitnami/kafka/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kafka
-version: 11.4.0
+version: 11.4.1
```

Comment:
Please bump the minor version instead

Reply:
A "minor" comment to bump the "minor" version? ;-)

Done!